src/models.py

```diff
@@ -98,8 +98,12 @@ def __init__(self, model, client):
                 if callable(attr):
                     self.should_index = attr
                 else:
-                    raise AlgoliaIndexError('{} should be a callable.'.format(
-                        self.should_index))
+                    should_index_attr_name = self.should_index
+                    if isinstance(attr, bool):
```

Comment:
you could add a test on a `@property` or a `BooleanField` and you should see it fail:
```python
# BooleanField
class Example(models.Model):
    index_me = models.BooleanField(default=False)

# or @property
class Example(models.Model):
    stuff = True

    @property
    def index_me(self):
        return self.stuff
```